modules/op-pipelines-as-code-splitting-logs-by-namespace.adoc

```diff
@@ -0,0 +1,14 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: REFERENCE
+[id="pipelines-as-code-splitting-logs-by-namespace_{context}"]
+= {pac} splitting logs by namespace
+
+The logs contain the namespace information to make it possible to grep logs on namespace or split the logs by a particular namespace. For example, to view the logs related to the `mynamespace` namespace, enter the following command:
```

Comment:
I am not sure "to grep" is good terminology for product documentatin - suggest "to filter"